pyrightconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This shouldn't be here. Please add a .gitignore file and include this file there.